circuits/ts/__tests__/TallyVotes.test.ts

```diff
@@ -261,7 +261,7 @@ describe('TallyVotes circuit', () => {
             poll.messageAq.merge(treeDepths.messageTreeDepth)
 
             for (let i = 0; i < NUM_BATCHES; i ++) {
-                poll.processMessages()
+                poll.processMessages(pollId)
```

Comment:
Why this change? Seems unrelated to the PR

Reply:
Unfortunately, this repo is still at a stage where you touch one little thing, and 1000 things break. I'll give you an example, you remove a console.log from the cli, and all integration tests fail 😓 
In this case, I added a declaration on the tsconfig file to generate type declarations -> this was done on the root tsconfig (which every package tsconfig extends) thus affecting the whole repo -> result is that there are quite a few mistakes in the codebase which just work, but Typescript is now complaining due to the type declarations being added

Reply:
Got it, appreciate the explanation!